lte/gateway/python/magma/subscriberdb/store/onready.py

```diff
@@ -27,7 +27,7 @@ class OnDataReady:
 
     def __init__(self, loop=None):
         self.loop = loop if loop else asyncio.new_event_loop()
-        self.event = asyncio.Event(loop=self.loop)
```

Comment:
This seems dangerous (here and other parts). If we no longer put the created event loop into the various asyncio objects, we probably shouldn't create and cache it either and just use `get_event_loop` when we need a loop.

I think this is a pretty good explanation of this deprecation warning: https://stackoverflow.com/a/60315290